compiler/native/compile_test.go

```diff
@@ -1475,16 +1499,25 @@ func Test_client_modifyConfig(t *testing.T) {
 			Commands:    []string{"echo hello from modification"},
 		})
 		output.Steps = steps
-		c.JSON(http.StatusOK, output)
+		response, err := convertResponse(want)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{"error": err.Error()})
+		}
+
+		c.JSON(http.StatusOK, response)
```

Comment:
small nitpick, should this newline be removed to ensure whitespace is consistent 
```suggestion
		c.JSON(http.StatusOK, response)
```

Reply:
Fixed.